python/adelphi/tests/integration/test_nb.py

```diff
@@ -0,0 +1,67 @@
+import glob
+import logging
+import os
+import sys
+import yaml
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+if os.name == 'posix' and sys.version_info[0] < 3:
+    import subprocess32 as subprocess
+else:
+    import subprocess
+
+from tests.integration import SchemaTestCase, setupSchema, dropKeyspace
+
+log = logging.getLogger('adelphi')
+
+NB_SCHEMA_PATH = "tests/integration/resources/nb-base-schema.cql"
+NB_REFERENCE_SCHEMA_PATH = "tests/integration/resources/nb-schemas/{}.yaml"
+
+class TestNb(SchemaTestCase):
+
+    # ========================== Unittest infrastructure ==========================
+    @classmethod
+    def setUpClass(cls):
+        setupSchema(NB_SCHEMA_PATH)
+
+    @classmethod
+    def tearDownClass(cls):
+        log.warning("Dropping keyspace testkeyspace")
+        dropKeyspace("testkeyspace")
```

Comment:
Re: the conversation above... this test has no dependency upon particular intstances to setup it's schema so it's safe to do so via class methods.  There's no need to do it one way or the other... the decision is entirely about what's good for a given test suite.